fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/query/GroupByQuerySpec.groovy

```diff
@@ -145,23 +145,27 @@ class GroupByQuerySpec extends Specification {
         vars.dataSource = vars.dataSource ?: '{"type":"table","name":"table_name"}'
         vars.granularity = vars.granularity ?: '{"type":"period","period":"P1D"}'
         vars.dimensions = vars.dimensions ?: "[]"
-        vars.filter = vars.filter ? ((' "filter": ').replaceAll(/\s/, "") + vars.filter + ',') : ""
-        vars.context = vars.context ? (('{"queryId":"dummy100",').replaceAll(/\s/, "") + vars.context + '}') : '{"queryId":"dummy100"}'
+        vars.filter = vars.filter ? /"filter": $vars.filter,/ : ""
+        vars.context = vars.context ?
+                /{"queryId":"dummy100",$vars.context}/ :
+                /{"queryId": "dummy100"}/
         vars.aggregations = vars.aggregations ?: "[]"
         vars.postAggregations = vars.postAggregations ?: "[]"
         vars.intervals = vars.intervals ?: "[]"
 
-        ("""{
-                "queryType":"$vars.queryType",
-                "dataSource":$vars.dataSource,
-                "granularity":$vars.granularity,
-                "dimensions":$vars.dimensions,
-                $vars.filter
-                "aggregations":$vars.aggregations,
-                "postAggregations":$vars.postAggregations,
-                "intervals":$vars.intervals,
-                "context":$vars.context
-            }""").replaceAll(/\s/, "")
+        """
+        {
+            "queryType":"$vars.queryType",
+            "dataSource":$vars.dataSource,
```

Comment:
I think it's because queryType is a String whereas others are objects which don't need to be quoted